### ISSUE TITLE
No app icon in Edge: sandstorm-pkgdef.capnp declares that jpegs are actually pngs

### ISSUE THREAD

**Issue by zarvox** [mnutt/davros#72]
Sandstorm serves app icons from its static endpoint.  It believes the content-type that was specified by the package metadata, and does not perform content-sniffing.

There are some JPEG files that are included in the pkgdef but are [described as PNGs](https://github.com/mnutt/davros/blob/master/.sandstorm/sandstorm-pkgdef.capnp#L45..L52).  This results in Sandstorm serving the bytes of a JPEG with a `Content-Type: image/png`.

Most browsers seem to ignore the Content-Type header and sniff the filetype of the content, and wind up showing .  However, Microsoft Edge (and possibly IE?) trust that the header the browser sent is accurate, and fail to render the app icon at all.

**Comment by mnutt** [comment]
Good catch, thanks. I'll make sure this gets fixed in the next release.

**Comment by ocdtrekkie** [comment]
This seems to still be unfixed, although the new Edge is based on Chromium, so people are unlikely to experience this issue.
